indico/modules/events/papers/models/call_for_papers.py

```diff
@@ -120,3 +120,7 @@ def user_competences(self):
     @property
     def announcement(self):
         return MarkdownText(paper_reviewing_settings.get(self.event, 'announcement'))
+
+    @property
+    def paper_reviewing_settings(self):
```

Comment:
Not sure if this makes sense. It's not specific to any event. IF you want to do this, then return `paper_reviewing_settings.bind(self.event)` - that will let you access the settings without passing the event again to get(), set() etc.